src/pages/coaching.tsx

```diff
@@ -66,62 +68,48 @@ const CoachingPage: FC<PageProps<Props, { language: string }>> = (props) => {
       <Container>
         <Title>{t("COACHING.TITLE")}</Title>
         <Subtitle>{t("COACHING.SUBTITLE")}</Subtitle>
+
         <CoverPhotoContainer>
           <Cover fluid={coachingCover.childImageSharp.fluid} />
         </CoverPhotoContainer>
+
         <P>{t("COACHING.INTRODUCTION")}</P>
         <H3>{t("COACHING.HOW_IT_WORKS")}</H3>
         <P>{t("COACHING.HOW_IT_WORKS_TEXT")}</P>
+
         <LessonHighlights language={language} />
+
         <H2>{t("COACHING.WEEKS")}</H2>
         <P>{t("COACHING.WEEKS_TEXT")}</P>
 
-        {/* <Weeks>
-          {weeks.map(({ node: week }: { node: ContentfulWeek }) => {
-            const bookmarked = data?.data.listLikedContents.items.find(
-              (item) => item.slug == week.slug
+        <Weeks>
+          {weeks.map((week: ContentfulWeek) => {
+            const bookmarkSearch = bookmarkedData?.find(
+              (item: any) => item.slug == week.slug
             )
+
+            bookmarkSearch === undefined
```

Comment:
This should be unnecessary. The returned weeks array contains objects already have the value whether they've bookmarked or not

Reply:
Also, you're defining the bookmarked value outside the map, which means that it will always reassign as the loop runs. It should also be constant like const bookmarked = bookmarkeSearch ? true : false, instead of reassigned variable. But just remove this because it should not be needed.

Reply:
In your opinion, what's the best approach to show the bookmarked content on the cards (week/lesson). We have two data sets. The `weeks` array and the `bookmarkedData` array. I took the inner loop away, so it's just the weeks array getting mapped. I'm just wondering what you think is the best way.

```javascript 
<Weeks>
          {weeks.map((week: ContentfulWeek) => {
            return (
              <WeekCard
                bookmarked={false}
                key={`${week?.slug}`}
                path={`/week/${week?.slug}`}
                intro={week?.intro}
                name={week?.weekName}
                duration={week?.duration}
                lessons={week?.lessons}
                coverPhoto={week?.coverPhoto?.fluid as FluidObject}
                slug={week.slug}
              />
            )
          })}
        </Weeks>```